docs/generate_summary.sh

```diff
@@ -0,0 +1,13 @@
+nu -c "help commands | get name | to md | save _names.md"
+
+path="docs/summaries.md"
+echo "" > $path
+while IFS= read -r command; do
+    echo "Appending usage for $command"
+    # Keep newlines when using command substition
+    RESULTX="$(nu -c "$command --help"; echo x)"
```

Comment:
What am I doing wrong?
![image](https://user-images.githubusercontent.com/343840/86938496-1a188300-c106-11ea-9392-1753312b5a4f.png)

Reply:
I see what's going on, there are two empty lines in the _names.md and the script is trying to find and command with an empty string. `nu -c "help commands | get name | to md | trim | save _names.md"` removes one of the empty lines. I guess the script should check if $command is empty first.

Reply:
Yeah, I don't have the script failing on error, so it still works ok.